setup.py

```diff
@@ -28,6 +28,6 @@
     long_description_content_type="text/markdown",
     url="https://github.com/ray-project/tune-sklearn",
     install_requires=[
-        "scikit-learn", "scipy", "ray[tune]>=2.0.0", "numpy>=1.16"
+        "scikit-learn", "scipy", "ray[tune]>=2.7.1", "numpy>=1.16"
```

Comment:
Does everything work with 2.7.1 or should we bump to 2.8 now?

Reply:
2.7.1 is an ok lower bound